conf/.htaccess.assets.cache

```diff
@@ -0,0 +1,13 @@
+# BEGIN core-standards:assets-cache
+<IfModule mod_headers.c>
+  # Client-side caching for assets with version query string.
+  # Not applied locally as Git hash does not change while editing.
+  <FilesMatch "\.(?:css|cur|eot|gif|gz|htc|ico|jpe?g|js|mp4|ogg|ogv|otf|png|svgz?|ttf|webm|woff2?)">
+    <If "%{HTTP_HOST} =~ /(local|nest)/">
+    </If>
+    <ElseIf "%{QUERY_STRING} =~ /ver=/">
+      Header set Cache-Control "public, max-age=31536000"
+    </ElseIf>
```

Comment:
Nested conditions should work, no?

Also added the delimiter ?& before ver.
```suggestion
    <If "%{HTTP_HOST} =~ /(local|nest)/">
      <If "%{QUERY_STRING} =~ /[?&]ver=/">
        Header set Cache-Control "public, max-age=31536000"
      </If>
    </If>
```

Reply:
- I've been testing all possible combinations on `netz-stage` and, as commented in Asana, nested <If> directives do not work, this is why I used this <ElseIf> workaround, which does work even if it is not the prettiest

- as for `QUERY_STRING`, the usual case is something `ver=d5e4f753`, which would not match `/[?&]ver=/` (I tested it).

Reply:
Quickly tested on https://stage.netz.netz.rocks/, this works:
```suggestion
    <If "%{HTTP_HOST} !~ /\.(local|nest)$/ && %{QUERY_STRING} =~ /(^|&)ver=/">
      Header set Cache-Control "public, max-age=31536000"
    </If>
```
If expressions support logical AND and OR conditions. See https://httpd.apache.org/docs/trunk/expr.html

Also fixed the comparison operator for the host name.

Of course, QUERY_STRING only contains the query string itself without its delimiter from the URL, so the ? is not part of it.  It is replaced with the start of string anchor ^.  However, that is not an actual character, so the character range [] cannot be used and needs to be replaced with multiple subgroups (|).

You can temporarily add `|rocks` to confirm that the domain exclusion works.